src/components/common/button/button.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from "react";
+import styles from "./styles.module.scss";
+
+type Props = {
+    name: string;
+    onClick: () => void;
+    width?: number | string;
+    height?: number | string;
+};
+
+const Button: FC<Props> = ({ name, onClick, width, height }) => (
+    <button
+        className={styles.button}
+        name={name}
+        onClick={onClick}
+        style={{ width, height }}
```

Comment:
is this necessary? maybe just add a classname property and combine it with button sty;e

Reply:
also fixed height/width doesn't seem to be a good practice)